pkg/filters/oidcadaptor/oidcadaptor.go

```diff
@@ -223,8 +223,10 @@ func (o *OIDCAdaptor) initDiscoveryOIDCConf() {
 		}
 	}
 	jwks, err := keyfunc.Get(oidcConf.JwksUri, keyfunc.Options{
-		Client:          httpCli,
-		RefreshInterval: interval,
+		Client:            httpCli,
+		JWKUseNoWhitelist: true,
+		RefreshInterval:   interval,
+		ResponseExtractor: keyfunc.ResponseExtractorStatusAny,
```

Comment:
The new options here, `JWKUseNoWhitelist` and `ResponseExtractor` are 100% backwards compatible fields. I will remove these fields in an upcoming commit to this PR.

`JWKUseNoWhitelist` is set to `true` which means that any `use` JWK parameter is valid. The project has since changed it's default to only allow those with a `use` of `sig`. See [more discussion here](https://github.com/MicahParks/keyfunc/releases/tag/v1.5.1).

`ResponseExtractor` is set to `keyfunc.ResponseExtractorStatusAny` which means that any HTTP response status code will not produce an error. The project has since changed it's default to only allow `http.StatusOK` to not produce an error. See [more discussion here](https://github.com/MicahParks/keyfunc/releases/tag/v1.4.0).